// nova-colors/lib/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

var colorValues = {
  colors: {
    cyan: '#7FC1CA',
    blue: '#83AFE5',
    purple: '#9A93E1',
    pink: '#D18EC2',
    red: '#DF8C8C',
    orange: '#F2C38F',
    yellow: '#DADA93',
    green: '#A8CE93'
  },
  grays: {
    gray0: '#1E272C',
    gray1: '#3C4C55',
    gray2: '#556873',
    gray3: '#6A7D89',
    gray4: '#899BA6',
    gray5: '#C5D4DD',
    gray6: '#E6EEF3'
  }
};

exports.default = colorValues;
var uiGroups = exports.uiGroups = _extends({
  userActionNeeded: colorValues.colors.red,
  userCurrentState: colorValues.colors.cyan,
  backgroundShade: colorValues.grays.gray0,
  background: colorValues.grays.gray1,
  foreground: colorValues.grays.gray5
}, colorValues.grays);

var syntaxGroups = exports.syntaxGroups = {
  constant: colorValues.colors.cyan,
  identifier: colorValues.colors.blue,
  statement: colorValues.colors.yellow,
  type: colorValues.colors.green,
  global: colorValues.colors.purple,
  emphasis: colorValues.colors.pink,
  special: colorValues.colors.orange,
  trivial: colorValues.grays.gray4
};

var versionControlGroups = exports.versionControlGroups = {
  added: colorValues.colors.green,
  modified: colorValues.colors.orange,
  removed: colorValues.colors.red,
  renamed: colorValues.colors.blue
};

var ansiGroups = exports.ansiGroups = {
  normal: {
    black: uiGroups.background,
    red: colorValues.colors.red,
    green: colorValues.colors.green,
    yellow: colorValues.colors.yellow,
    blue: colorValues.colors.blue,
    magenta: colorValues.colors.purple,
    cyan: colorValues.colors.cyan,
    white: uiGroups.foreground
  },
  bright: {
    black: colorValues.grays.gray4,
    red: colorValues.colors.orange,
    green: colorValues.colors.green,
    yellow: colorValues.colors.yellow,
    blue: colorValues.colors.blue,
    magenta: colorValues.colors.pink,
    cyan: colorValues.colors.cyan,
    white: colorValues.grays.gray6
  }
};
